updater/osv/osv.go

```diff
@@ -209,8 +178,62 @@ func (u *updater) Fetch(ctx context.Context, fp driver.Fingerprint) (io.ReadClos
 			Msg("error closing ecosystems.txt response body")
 	}
 	if err != nil {
-		return nil, fp, err
+		return s, err
+	}
+
+	return s, nil
+}
+
+// Ignore is a set of incoming ecosystems that we can throw out immediately.
+var ignore = map[string]struct{}{
+	"alpine":         {}, // Have a dedicated alpine updater.
+	"android":        {}, // AFAIK, there's no Android container runtime.
+	"debian":         {}, // Have a dedicated debian updater.
+	"github actions": {}, // Shouldn't be in containers?
+	"linux":          {}, // Containers have no say in the kernel.
+	"oss-fuzz":       {}, // Seems to only record git revisions.
+}
+
+type updater struct {
+	name      string
+	ecosystem string
+	c         *http.Client
+	uri       *url.URL
+}
+
+func (u *updater) Name() string { return u.name }
+
+type UpdaterConfig struct {
+	// The URL serving data dumps behind an S3 API.
```

Comment:
I need to circle back and fix these references to the S3 api that aren't true anymore.